docs/reference/indices/put-mapping.asciidoc

```diff
@@ -173,8 +173,17 @@ PUT my_index/_mapping/type_one <2>
     }
   }
 }
+-----------------------------------
+// AUTOSENSE
+// TEST[catch:request]
+<1> Create an index with two types, both of which contain a `text` field which have the same mapping.
+<2> Trying to update the `search_analyzer` just for `type_one` throws an exception like `"Merge failed with failures..."`.
+
+But this then running this succeeds:
```

Comment:
Here I broke the snippet up because I couldn't easily cram it into the (arbitrary) rules I made for the tests.